guides/doc-Installing_Proxy/master.adoc

```diff
@@ -26,26 +26,9 @@ include::common/assembly_installing-capsule-server.adoc[leveloffset=+1]
 
 include::common/assembly_performing-additional-configuration-on-smart-proxy-server.adoc[leveloffset=+1]
 
-// Configuring {SmartProxyServer} with External Services
-include::common/assembly_configuring-external-services.adoc[leveloffset=+1]
-
-include::common/assembly_managing-dhcp-on-smart-proxies.adoc[leveloffset=+1]
-
-include::common/assembly_managing-dns-on-smart-proxies.adoc[leveloffset=+1]
-
-include::common/assembly_using-infoblox-as-dhcp-and-dns-providers.adoc[leveloffset=+1]
-
 :numbered!:
 
 // {SmartProxyServer} Scalability Considerations
 [appendix]
 include::common/modules/ref_smart-proxy-server-scalability-considerations-when-managing-puppet-clients.adoc[leveloffset=+1]
 
-ifndef::satellite[]
-[appendix]
-include::common/modules/ref_dhcp-isc-settings.adoc[leveloffset=+1]
-
-[appendix]
-include::common/modules/ref_dhcp-options-for-network-configuration.adoc[leveloffset=+1]
-endif::[]
-endif::[]
```

Comment:
Looks like you remove one "endif" too much. Unless this guide is now ready, then we could also remove lines 10-16. Probably a question for Ewoud.

Reply:
good catch. I added the `endif` statement again.